includes/Dashboard/Templates/Settings.php

```diff
@@ -53,16 +53,18 @@ public function show_enable_seller_message() {
      */
     public function render_settings_header() {
         global $wp;
+        $is_main_setting = 0;
```

Comment:
Variable name `is_store_setting` is more relevant. Please use true or false for boolean.